### PR TITLE
Add fork tests for vote casting

### LINKED ISSUE #880: Add happy path integration fork tests for governace implementation

### PR DESCRIPTION
**Pull request by mjlescano**
Closes #880

### CODE REVIEW

File: packages/spartan-council/.env.sample [hunk 1]
```diff
@@ -1,11 +1 @@
...
-# -------------------------------------------------------------------------------
-DEPLOYER_PRIVATE_KEY=
-# -------------------------------------------------------------------------------
-
-# -------------------------------------------------------------------------------
-# INFURA PROVIDER
-# -------------------------------------------------------------------------------
-INFURA_KEY=
-# -------------------------------------------------------------------------------
+FORK_PROVIDER_URL=https://opt-mainnet.g.alchemy.com/v2/<api-key>
```
Comment (leomassazza) [inline packages/spartan-council/.env.sample:1]: is this intentional?

Comment (mjlescano) [reply packages/spartan-council/.env.sample:1]: Yup, we need that env var for executing integration tests, and we don't need the previously declared vars

Comment (mjlescano) [reply packages/spartan-council/.env.sample:1]: Sorry, you were right, we need `DEPLOYER_PRIVATE_KEY`, re adding

**Review by leomassazza — APPROVED** [review]

File: packages/spartan-council/.env.sample (hunk 1 above)
Comment (leomassazza) [reply packages/spartan-council/.env.sample:9]: 👌

**Pull request merged**